.github/workflows/go-test.yaml

```diff
@@ -0,0 +1,23 @@
+name: go-test
+on:
+  push:
+    branches:
+      - "!main"
+  workflow_call:
+  pull_request:
+jobs:
```

Comment:
You could add some spacing for better readability between the main sections. Is there pretty formatting available for GHA?

Reply:
added